modules/gltf/extensions/physics/gltf_physics_body.h

```diff
@@ -33,27 +33,47 @@
 
 #include "scene/3d/physics_body_3d.h"
 
-// GLTFPhysicsBody is an intermediary between OMI_physics_body and Godot's physics body nodes.
+// GLTFPhysicsBody is an intermediary between Godot's physics body nodes
+// and the OMI_physics_body extension.
 // https://github.com/omigroup/gltf-extensions/tree/main/extensions/2.0/OMI_physics_body
 
 class GLTFPhysicsBody : public Resource {
 	GDCLASS(GLTFPhysicsBody, Resource)
 
+public:
+	// These values map to Godot's physics body types.
+	// When importing, the body type will be set to the closest match, and
+	// user code can change this to make Godot generate a different node type.
+	// When exporting, this will be squashed down to one of "static",
+	// "kinematic", or "dynamic" motion types, or the "trigger" property.
+	enum class PhysicsBodyType {
+		STATIC,
+		ANIMATABLE,
+		CHARACTER,
+		RIGID,
+		VEHICLE,
+		TRIGGER,
+	};
+
 protected:
 	static void _bind_methods();
 
 private:
-	String body_type = "static";
+	PhysicsBodyType body_type = PhysicsBodyType::RIGID;
 	real_t mass = 1.0;
 	Vector3 linear_velocity;
 	Vector3 angular_velocity;
 	Vector3 center_of_mass;
-	Basis inertia_tensor = Basis(0, 0, 0, 0, 0, 0, 0, 0, 0);
+	Vector3 inertia_diagonal;
+	Quaternion inertia_orientation;
 
 public:
 	String get_body_type() const;
 	void set_body_type(String p_body_type);
 
+	PhysicsBodyType get_physics_body_type() const;
+	void set_physics_body_type(const PhysicsBodyType p_body_type);
```

Comment:
```suggestion
	void set_physics_body_type(PhysicsBodyType p_body_type);
```
No `const` on simple types